openwisp-utils-qa-checks

```diff
@@ -42,36 +43,45 @@ runcheckendline() {
     fi
   done
   if [ $flag -ne 0 ]; then
-    exit 1
+    echoerr "ERROR: Blank endline check failed!"
+    FAILURE=1
+  else
+    echo "SUCCESS: Blank endline check successful!"
   fi
 }
 
 runcheckmigrations() {
-  echo "Running migration name check"
   if [ "$MIGRATION_PATH" == "" ]; then
-    echo "No migration path specified!"
-    exit 5
+    echoerr "No migration path specified!"
+    FAILURE=1
+    return
   fi
-  checkmigrations --migration-path "$MIGRATION_PATH" --migrations-to-ignore "$MIGRATIONS_TO_IGNORE" || exit 2
+  checkmigrations --migration-path "$MIGRATION_PATH" --migrations-to-ignore "$MIGRATIONS_TO_IGNORE" \
+    && echo "SUCCESS: Migration name check successful!" \
+    || { echoerr "FAIL: Migration name check failed!"; FAILURE=1; }
 }
 
 runflake8() {
-  echo "Running flake8 check"
-  flake8 || exit 3
+  flake8 && echo "SUCCESS: Flake8 check successful!" \
+    || { echoerr "FAIL: Flake8 check failed!"; FAILURE=1; }
 }
 
 runisort() {
-  echo "Running isort"
-  isort --check-only --recursive --diff || exit 4
+  isort --check-only --recursive --diff \
+    && echo "SUCCESS: Isort check successful!" \
+    || { echoerr "FAIL: Isort check failed!"; FAILURE=1; }
 }
 
 runcheckcommit() {
-  echo "Running checkcommit"
   [ -n "$TRAVIS_PULL_REQUEST" ] || TRAVIS_PULL_REQUEST="false"
   if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
     printf "Checking commit message:\n\n"
     printf "%s\n\n" "$COMMIT_MESSAGE"
```

Comment:
Can you please move it inside the `FAIL` condition only? 

Why? (It's a nitpick really!)
- Because all the other tests are giving additional output only when there is a failure, not with success, so I think we should keep it uniform.
- Unix philosophy of programming which says that the program shouldn't give an output to user unless the output is required and I think that in success case, I don't need to see the commit message, that's all!